cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This example file is regenerated if the fixtures directory is empty. We could alternately add it to .gitignore.